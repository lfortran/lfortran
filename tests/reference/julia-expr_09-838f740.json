{
    "basename": "julia-expr_09-838f740",
    "cmd": "lfortran --no-color --show-julia {infile}",
    "infile": "tests/../integration_tests/expr_09.f90",
    "infile_hash": "3b99f849ef5412d7fbc16bc164307d14964150e05dd203596a3e6dd8",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "julia-expr_09-838f740.stdout",
    "stdout_hash": "d7b9f6149acd732a2852e617b0aed1cc4484ab715db3b3cfbd644c09",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}