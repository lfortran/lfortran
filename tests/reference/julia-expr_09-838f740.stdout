function main()
    local x4::Int32 = 0
    local x8::Int64 = 0
    local y4::Int32 = 0
    local y8::Int64 = 0
    y4 = 5
    x4 = y4 ^ 2
    println(y4, " ", x4)
    if x4 ≠ 25
        println(Base.stderr, "ERROR STOP")
        exit(1)
    end
    y8 = 5
    x8 = y8 ^ 2
    println(y8, " ", x8)
    if x8 ≠ 25
        println(Base.stderr, "ERROR STOP")
        exit(1)
    end
end

main()
